model.py

```diff
@@ -50,6 +50,24 @@ def __init__(self, file_name:str):
                 result[date].append(word_obj)
 
         self.dictionary = result 
+    
```

Comment:
In tutte le funzioni che prendono in ingresso la data mi chiedo se ha senso prenderla come ingresso. Se io vedo una parola perché l'ho estratta non so a che data sia corrispondente però magari voglio eliminarla. Quindi non posso dargli in ingresso la data. 
Al contrario se sto aggiungendo parole su oggi e mi accorgo di aver sbagliato so che la data è oggi quindi posso dargliela in ingresso, però è l'unico caso

Reply:
Si ha senso perchè le funzioni sono generiche il fatto che tu lo sappia non fa si che il computer lo sappia

Reply:
"Se hai estratto una parola e non sai che data è" è falso sai sempre a che data è perchè le funzioni che tornano una parola random tornano apposta anche la data anche per quel motivo.

Reply:
Non so sempre a che data è. Perché le funzioni che estraggono una parola a 3 60 90 g fa non è detto che prendano davvero 30 60 e 90 g fa. Estraggono la prima parola non vuota che trovano partendo da 30g fa. quindi in quel caso non ho la data. Si potrebbe stamparla volendo così si risolve il problema